packages/components-native/src/Typography/Typography.style.ts

```diff
@@ -0,0 +1,362 @@
+import { StyleSheet, TextStyle } from "react-native";
+import { tokens } from "../utils/design";
+
+const extravagantLineHeight = tokens["typography--lineHeight-extravagant"];
+const jumboLineHeight = tokens["typography--lineHeight-jumbo"];
+const largestLineHeight = tokens["typography--lineHeight-largest"];
+const largerLineHeight = tokens["typography--lineHeight-larger"];
+const largeLineHeight = tokens["typography--lineHeight-large"];
+const baseLineHeight = tokens["typography--lineHeight-base"];
+const tightLineHeight = tokens["typography--lineHeight-tight"];
+const minisculeLineHeight = tokens["typography--lineHeight-miniscule"];
+
+/**
+ * `StyleSheet` for Typography.tsx.
+ *
+ * If you find yourself needing to use what's inside this object on files other
+ * than `<Typography />`, please import from `@jobber/components-native` instead.
+ *
+ * ```
+ * import { typographyStyles } from "@jobber/components-native"
+ * ```
+ */
+
+/**
+ * Reusable typography tokens to ensure consistency for any client facing texts.
+ */
+export const typographyStyles: { [index: string]: TextStyle } =
+  StyleSheet.create({
+    // This follows a pattern of
+    // { fontFamily }{ fontStyle }{ fontWeight }
+    baseRegularRegular: {
+      fontFamily: "inter-regular",
+    },
```

Comment:
Oh, you do need to export this on its own const like before since it's not gonna work for web and storybook due to some fun react native performance thing

Reply:
now that I think about it, maybe this should be under utils (typographyTokens) since it's shared through a lot of other components

Reply:
So, we need to export both of them? The object with the values and the const with `Stylesheet.create`?